go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/diagridio/go-etcd-cron
 
-go 1.21.9
```

Comment:
Can we postpone the change to Go 1.22 or is it required because of the dependency on kit?

Reply:
I just had to add toolchain to pin the patch version too.